libc/src/__support/CPP/new.cpp

```diff
@@ -16,15 +16,29 @@ void operator delete(void *mem, std::align_val_t) noexcept { ::free(mem); }
 void operator delete(void *mem, size_t) noexcept { ::free(mem); }
 
 void operator delete(void *mem, size_t, std::align_val_t) noexcept {
+#ifdef LIBC_TARGET_OS_IS_WINDOWS
```

Comment:
I guess relying on transitive inclusion of src/__support/macros/properties/os.h is fine in this case.